Makefile

```diff
@@ -297,22 +308,35 @@ release/bundle-image:
 	bazel run --stamp --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 \
 		//:push_operator_bundle_image
 
-
-OLM_REPO ?=
-OLM_BUNDLE_REPO ?= cockroachdb-operator-index
-OLM_PACKAGE_NAME ?= cockroachdb-certified
-TAG ?= $(APP_VERSION)
+# This target:
+# 1. Updates CRD and CSV files 
+# 2. Pushes the operator image to a registry
+# 3. Builds the OpenShift bundle
+# 4. Pushes the OpenShift bundle to a registry
+# 5. Run opm to create and push the OpenShift index container to a registry
+# 6. Removes the newly created OpenShift files so that it can run again. 
 #
-# dev opm index build for quay repo
+# The following env variables are used for the above process.
 #
-.PHONY: dev/opm-build-index
-dev/opm-build-index:
-	RH_BUNDLE_REGISTRY=$(RH_BUNDLE_REGISTRY) \
-	RH_BUNDLE_IMAGE_REPOSITORY=$(RH_BUNDLE_IMAGE_REPOSITORY) \
-	RH_BUNDLE_VERSION=$(RH_BUNDLE_VERSION) \
-	RH_DEPLOY_PATH=$(RH_DEPLOY_FULL_PATH) \
+# APP_VERSION
+# VERSION
+# RH_BUNDLE_VERSION
+# RH_OPERATOR_IMAGE
+# DOCKER_REGISTRY
+#
+# See hack/openshift-test-packaging.sh for more information on running this target.
+.PHONY: test/openshift-package
+test/openshift-package: release/update-pkg-manifest release/image release/opm-build-bundle test/push-openshift-images
+	VERSION=$(VERSION) \
+	hack/cleanup-packaging.sh 
+
+# This target pushes the OpenShift bundle, then uses opm to push the index bundle.
+.PHONY: test/push-openshift-images
+test/push-openshift-images:
+	APP_VERSION=$(APP_VERSION) \
+	DOCKER_REGISTRY=$(DOCKER_REGISTRY) \
 	bazel run --stamp --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 \
-		//hack:opm-build-index $(OLM_REPO) $(OLM_BUNDLE_REPO) $(TAG) $(RH_BUNDLE_VERSION) $(RH_COCKROACH_DATABASE_IMAGE)
```

Comment:
The opm-build-index bazel target was a dev target that never worked.  I have replaced it with a target that works, and could do the release for us as well.